src/renderer/processes/multisigs/lib/mulitisigs-utils.ts

```diff
@@ -1,43 +1,85 @@
-import { AccountType, type Chain, ChainOptions, ChainType, CryptoType, SigningType, WalletType } from '@/shared/core';
+import {
+  type AccountId,
+  AccountType,
+  type Chain,
+  ChainOptions,
+  ChainType,
+  CryptoType,
+  type FlexibleMultisigAccount,
+  type MultisigAccount,
+  type NoID,
+} from '@/shared/core';
 import { isEthereumAccountId, toAddress } from '@/shared/lib/utils';
 
 export const multisigUtils = {
   isMultisigSupported,
-  buildMultisig,
+  isFlexibleMultisigSupported,
+  buildMultisigAccount,
+  buildFlexibleMultisigAccount,
 };
 
-function isMultisigSupported(chain: Chain): boolean {
-  return Boolean(chain.options?.includes(ChainOptions.MULTISIG));
+function isMultisigSupported(chain: Chain) {
+  return chain.options?.includes(ChainOptions.MULTISIG) ?? false;
+}
+
+function isFlexibleMultisigSupported(chain: Chain) {
+  return isMultisigSupported(chain) && chain.options?.includes(ChainOptions.PURE_PROXY);
```

Comment:
As I recall it can be any proxy not only pure for multisig to be flexible (but we create flexible only with pure)

Reply:
oh, okay, i'll fix it